core/deployment/src/main/java/io/quarkus/BannerConfig.java

```diff
@@ -1,10 +1,17 @@
-package io.quarkus.banner.deployment;
+package io.quarkus;
```

Comment:
Minor but could we still have an `io.quarkus.banner` package? I find it weird to have that one in the root package.

Reply:
Sure, no problem

Reply:
Fixed